x11-misc/shutter/shutter-0.94.3-r1.ebuild

```diff
@@ -0,0 +1,87 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+inherit xdg-utils desktop
+
+DESCRIPTION="Feature-rich screenshot program"
+HOMEPAGE="https://shutter-project.org/"
+#SRC_URI="https://shutter-project.org/wp-content/uploads/releases/tars/${P}.tar.gz"
+SRC_URI="https://launchpad.net/shutter/0.9x/${PV}/+download/${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="minimal"
+
+RDEPEND="dev-lang/perl
+	dev-perl/libxml-perl
+	dev-perl/gnome2-canvas
+	dev-perl/gnome2-perl
+	dev-perl/gnome2-wnck
+	dev-perl/Gtk2-Unique
+	dev-perl/Gtk2-ImageView
+	dev-perl/File-DesktopEntry
+	dev-perl/File-HomeDir
+	dev-perl/File-Which
+	dev-perl/JSON
+	dev-perl/File-Copy-Recursive
+	dev-perl/File-MimeInfo
+	dev-perl/Locale-gettext
+	dev-perl/Net-DBus
+	dev-perl/Proc-Simple
+	dev-perl/Proc-ProcessTable
+	dev-perl/Sort-Naturally
+	dev-perl/WWW-Mechanize
+	dev-perl/X11-Protocol
+	dev-perl/XML-Simple
+	dev-perl/libwww-perl
+	virtual/imagemagick-tools[perl]
+	!minimal? (
+		dev-perl/Goo-Canvas
+		dev-perl/JSON-MaybeXS
+		dev-perl/Net-OAuth
+		dev-perl/Path-Class
+		media-libs/exiftool
+	)
+"
+
+src_prepare() {
+	default
+
+	use minimal && eapply "${FILESDIR}"/${PN}-0.90-goocanvas.patch
```

Comment:
Since you need to patch off core features, maybe it'd be better to pull goo-canvas unconditionally and stop negatively patching shutter?